src/test/scala/org/camunda/feel/api/context/CustomContextTest.scala

```diff
@@ -18,65 +18,63 @@ package org.camunda.feel.api.context
-  VariableProvider
-}
+import org.camunda.feel.context.{CustomContext, FunctionProvider, VariableProvider}
 import org.camunda.feel.context.VariableProvider.StaticVariableProvider
+import org.camunda.feel.impl.{EvaluationResultMatchers, FeelEngineTest}
 import org.camunda.feel.syntaxtree._
 import org.scalatest.matchers.should.Matchers
 import org.scalatest.flatspec.AnyFlatSpec
 
-class CustomContextTest extends AnyFlatSpec with Matchers {
-
-  val engine = new FeelEngine
+class CustomContextTest extends AnyFlatSpec with Matchers with FeelEngineTest with EvaluationResultMatchers {
 
   "A default context" should "provide its members" in {
-    engine.evalExpression("a", variables = Map("a" -> 2)) should be(Right(2))
-    engine.evalUnaryTests(
-      "2",
-      variables = Map(UnaryTests.defaultInputVariable -> 2)) should be(
-      Right(true))
+    evaluateExpression(
+      expression = "a",
+      variables = Map("a" -> 2)
+    ) should returnResult(2)
+
+    evaluateUnaryTests(
+      expression = "2",
+      inputValue = 2
+    ) should returnResult(true)
   }
 
-  it should "fail on access to missing member" in {
-    engine.evalExpression("b", variables = Map("a" -> 2)) should be
-    Left(
-      Failure(
-        "failed to evaluate expression 'b': no variable found for name 'b'"))
+  it should "return null if the variable doesn't exist" in {
+    evaluateExpression(
+      expression = "b",
+      variables = Map("a" -> 2)
+    ) should returnNull()
   }
 
   "A custom context" should "provide its members" in {
     val myCustomContext = new CustomContext {
 
       override def variableProvider: VariableProvider = new VariableProvider {
         override def getVariable(name: String): Option[Any] = name match {
-          case "a"                             => Some(2)
+          case "a" => Some(2)
           case UnaryTests.defaultInputVariable => Some(2)
-          case _                               => None
+          case _ => None
         }
 
         override def keys: Iterable[String] =
           List("a", UnaryTests.defaultInputVariable)
       }
 
     }
-    engine.evalExpression("a", myCustomContext) should be(Right(2))
-    engine.evalExpression("floor(3.8)", myCustomContext) should be(Right(3))
-    engine.evalUnaryTests("2", myCustomContext) should be(Right(true))
-  }
 
-  it should "fail on access to missing member" in {
-    val context = new CustomContext {
-      override def variableProvider: VariableProvider =
-        VariableProvider.StaticVariableProvider(Map.empty)
-    }
+    engine.evaluateExpression(
+      expression = "a",
+      context = myCustomContext
+    ) should returnResult(2)
+
+    engine.evaluateExpression(
+      expression = "floor(3.8)",
+      context = myCustomContext) should returnResult(3)
 
-    engine.evalExpression("b", context) should be
-    Left(
-      Failure(
-        "failed to evaluate expression 'b': no variable found for name 'b'"))
+    engine.evaluateUnaryTests(
```

Comment:
💭 Maybe we can modify a bit the helper method `evaluateUnaryTests` by providing a custom context. So we can use it here